test/integration/stomp_/publisher/create_request_test.py

```diff
@@ -43,47 +21,71 @@ def system_spec():
     }
 
 
-class MessageListener(object):
-    create_event_captured = False
+class MessageListener(ConnectionListener):
```

Comment:
Since this is being passed to stomp, it makes sense that it should inherit from the type that stomp expects.